src/negotiation/maker_negotiator.ts

```diff
@@ -29,44 +35,58 @@ export class MakerNegotiator {
     return this.ordersById[orderId];
   }
 
-  public acceptOrder(
-    // @ts-ignore
-    order: Order
-  ): ExecutionParams | undefined {
+  public getExecutionParams(): ExecutionParams {
+    return this.executionParams;
+  }
+
+  public acceptOrder(swapId: string, order: Order) {
```

Comment:
❓ I don't understand why the swap ID is back. If I don't misunderstand something, we changed this to be by Order on purpose. The swap ID is an execution parameter, that the maker comes up with after the taker has taken the order. I am not we keep the boundaries correctly here. 
The `ComitClient` now exposes `retrieveSwapById(swapId: string)` again. If we call `acceptOrder(swapId: string, order: Order)` this implies we have to remember the `swapId` in the application, not just the Order. Is this what we want?

Reply:
This as per the explanation I gave you and Philipp about a negotiation protocol improvement when using the white board.

Reply:
Sorry, guess I forgot about that discussion 😅 - Go ahead then 😃